### DISCUSSION TITLE
api not working in postman (auth)

### DISCUSSION THREAD

**Discussion by ameerfayiz** [serverpod/serverpod#2402] [Q&A]
why is this post request not working now? it used to work in postman in earlier versions of serverpod. but now, its returning unauthorized error in 2.0.0 version. what's the format of  "auth" that i should give in 2.0.0 onwards ??

```
{   
   "method" : "updateAirportList",
   "auth": "{{keyId}}:{{key}}",
   "Airports": [..]
}
```

@vlidholt @SandPod @Isakdl @TJ-coding @Aygistov @letscodetpl @gstuntz @NeroSong @CordMemescape

Basically this is my intention:
I have to write a script to update list of rows to the database . Earlier, i wrote the script in python using above request format and it was working. But now i am getting 401 error . How can i do the same it in python or dart? Please reply ASAP. Urgent 🦺

**Comment by SandPod** [comment]
Replying to the reported issue.
